Yosemite/Yosemite/Stores/PaymentGatewayAccountStore.swift

```diff
@@ -50,11 +52,15 @@ public final class PaymentGatewayAccountStore: Store {
 
 // MARK: Networking Methods
 private extension PaymentGatewayAccountStore {
+    /// We support payment gateway accounts for both the WooCommerce Payments extension AND
+    /// the Stripe extension. Let's attempt to load each and update view storage with the results.
+    /// Call the passed completion after both loads have been attempted.
```

Comment:
nit: maybe these comments are worth mentioning in its action `PaymentGatewayAccountAction.loadAccounts` especially on the completion block behavior (always successful after both requests are made regardless of the state of their results)?

Reply:
Fixed in https://github.com/woocommerce/woocommerce-ios/pull/5734/commits/17b96bb0eb44f188f0f5d6e4f0878f162b59f068